docs/imaging_protocols/confocal/confocal_imaging.md

```diff
@@ -0,0 +1,52 @@
+
+# Correlative Confocal - QPI Imaging
+
+The microscope images of the markers were acquired using correlative confocal fluorescence microscopy and quantitative phase microscopy.
+
+## Sample preparation
+For fixed and live cell imaging 150,000 of A549 cells were plated in 24 well glass bottom plate.
+
+### Fixed samples:
+* The day after plating the cells, the nucleus was stained with Hoechst in 1:10,000 dilution in PBS for 10 mins.
+* Cells were fixed with either 4% of PFA or cold MtOH for 20 mins.
+* Add imaging media (water?)
```

Comment:
@Soorya19Pradeep  can you confirm with Sookie what is used for this sample. My other suggestion would be to just add the link to the protocol.io. here.